.github/scripts/runner_determinator.py

```diff
@@ -150,7 +154,7 @@ def get_workflow_type(issue: Issue, workflow_requestors: Iterable[str]) -> str:
             return WORKFLOW_LABEL_LF
         else:
             all_opted_in_users = {
-                usr_raw.strip("\n\t@ ") for usr_raw in first_comment.split()
+                usr_raw.strip("\n\t@ ").split(',')[0] for usr_raw in first_comment.split()
```

Comment:
I will update the commit message and PR description with the details of the userlist format.

I designed it to be backwards compatible with the current issue comment format so its still a newline separated list of users but with the addition of each user line being a comma-separated list of additional features or experiments they want to opt-in to. Since this change is about adding support for the Amazon Linux 2023 feature which I believe it achieves I think we should proceed without making major changes to the input format for now.

I saw your proposal to improve the runner determinator to allow include/exclude specific jobs which has a proposal for YAML format change I think we should do the change for that one along with that feature.